web/src/app/login/page.tsx

```diff
@@ -2,23 +2,41 @@ import CompanyLogo from "../components/company-logo";
 import LoginForm, { LoginFormSchema } from "./components/login-form";
 
 export default function LoginPage() {
+    // Parâmetro credentials tem a validação e o tipo de LoginFormSchema, que é um objeto com email e password
     async function login(credentials: LoginFormSchema) {
-        "use server";
+        // Tudo que estiver abaixo de "use server" será um server client
+        ("use server");
```

Comment:
`"use server"` não deve estar dentro de parênteses.
```suggestion
        "use server";
```